src/core/include/openvino/op/util/topk_base.hpp

```diff
@@ -18,29 +18,28 @@ class OPENVINO_API TopK_Base : public Op {
     OPENVINO_OP("TopK_Base", "util");
     TopK_Base() = default;
 
-    /// \param arg The node producing the input data batch tensor.
-    /// \param strides The strides.
-    /// \param pads_begin The beginning of padding shape.
-    /// \param pads_end The end of padding shape.
-    /// \param kernel The kernel shape.
-    /// \param rounding_mode Whether to use ceiling or floor rounding type while
-    /// computing output shape.
-    /// \param auto_pad The pad type for automatically computing padding sizes.
+    /// \brief The common base class for all TopK operator versions
+    ///
+    /// \param data The input tensor
+    /// \param k Specifies how many maximum/minimum elements should be computed
+    /// \param axis The axis along which TopK should be computed
+    /// \param mode Specifies whether the maximum or minimum elements are selected
+    /// \param sort Specifies the order of output elements and/or indices
+    ///             Accepted values: none, index, value
+    /// \param index_element_type Specifies the type of produced indices
     TopK_Base(const Output<Node>& data,
```

Comment:
What about TopK_Base -> TopKBase? ;)
```suggestion
    TopKBase(const Output<Node>& data,
```

Reply:
Sure, why not ;)